### PR TITLE
Added Snippet Generator config for MarathonStep

### PR DESCRIPTION
**Pull request by orevial**
Just added a config.jelly file to enable the Snippet Generator to generate a Groovy line with URL/Force Update/App Id/Docker args.
Generates Groovy as follows :
`marathon appid: 'myappid', docker: 'mydocker', forceUpdate: true, url: 'mymarathonurl'`

### CODE REVIEW

**Comment by KenanSulayman** [comment]
👍

**Comment by orevial** [comment]
Note : We'll need to add `forcePullImage `param once [PR 22](https://github.com/jenkinsci/marathon-plugin/pull/22) gets merged.

**Comment by colin-msphere** [comment]
Please use the `${%Title Text}` format for the `title` attributes as described on the [Internationalization page](https://wiki.jenkins-ci.org/display/JENKINS/Internationalization#Internationalization-Markingmessagesinjellyfiles).

**Comment by orevial** [comment]
Done, hope it's okay

**Comment by colin-msphere** [comment]
retest this please

File: src/main/resources/com/mesosphere/velocity/marathon/MarathonStep/config.jelly [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    <f:entry field="url" title="${%Marathon URL}">
+        <f:textbox/>
+    </f:entry>
+    <f:entry field="forceUpdate" title="${%Force Update}">
+        <f:checkbox name="forceUpdate" />
+    </f:entry>
+    <f:entry field="appid" title="${%App ID}">
+        <f:textbox/>
+    </f:entry>
+    <f:entry field="docker" title="${%Docker}">
```
Comment (colin-msphere) [inline src/main/resources/com/mesosphere/velocity/marathon/MarathonStep/config.jelly]: Please change the title to `${%Docker Image}`.

**Comment by colin-msphere** [comment]
Thank you for the pull request!

**Pull request merged**